charts/incubator/zammad/values.yaml

```diff
@@ -0,0 +1,80 @@
+image:
+  pullPolicy: IfNotPresent
+  repository: tccr.io/truecharts/zammad
+  tag: latest@sha256:c9c8ec07af85ca06ff2776e1bd00c70cca470c7bf9aa92c716b78283bf880f49
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsGroup: 0
+    runAsUser: 0
+
+service:
+  main:
+    enabled: false
+    ports:
+      main:
+        enabled: false
+
+zammad:
+  zammad:
+    zammad_autowizardjson: ""
+  webconcurrency:
+    webcon_instances: ""
+    webcon_sessionjobs: 0
+    webcon_schedprocesses: false
+    webcon_delayedjobs: 0
+  rails:
+    rails_trustedproxies: ['127.0.0.1', '::1']
+  redis:
+    redis_url: "http://redis.ix-redis.svc.cluster.local:6379"
+  memcached:
+    memcached_url: "http://memcached.ix-memcached.svc.cluster.local:11211"
```

Comment:
{{ printf "%v-%v:%v" .Release.Name "memcached" "11211" }}

Reply:
Commit Steven's proposed changes

Reply:
Got it, but had to use `'` instead of `"` for the enclosing quotes. (Shout out to stavros for showing me that playground website which helped!)